### PR TITLE
docs(traffic_light_classifier): fix typo

### PR DESCRIPTION
**Pull request by Shin-kyoto**
## Description

- I fixed typo.

## Tests performed

```
❯ cspell perception/traffic_light_classifier/README.md
1/1 ./perception/traffic_light_classifier/README.md 294.36ms
CSpell: Files checked: 1, Issues found: 0 in 0 files
```

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by miursh — APPROVED** [review]
LGTM
but, I'm just curious, why did you separate this PR from https://github.com/autowarefoundation/autoware.universe/pull/3355

**Comment by Shin-kyoto** [comment]
> LGTM but, I'm just curious, why did you separate this PR from #3355

I separated #3355 and #3367 because while it was certain that #3355 would be merged, perhaps there were discussions on whether to add cspell or ignore it inline(this PR).

File: perception/traffic_light_classifier/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<!-- cspell: ignore mmclassification mmcv -->
```
Comment (kenji-miyake) [inline perception/traffic_light_classifier/README.md:1]: I think you can add `MMCV` to the dictionary.
```suggestion
<!-- cspell: ignore mmclassification -->
```
https://github.com/open-mmlab/mmcv

Comment (kenji-miyake) [reply perception/traffic_light_classifier/README.md:1]: https://github.com/tier4/autoware-spell-check-dict/pull/505

Comment (Shin-kyoto) [reply perception/traffic_light_classifier/README.md]: I removed mmcv from ignore.
And thanks a lot for PR below. 
https://github.com/tier4/autoware-spell-check-dict/commit/c78b32e5dfa46f37fd7d414321ee93258c25b354

Comment (kenji-miyake) [reply perception/traffic_light_classifier/README.md]: And `mmclassification` seems to be also okay.
https://mmclassification.readthedocs.io/en/latest/getting_started.html

File: perception/traffic_light_classifier/README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<!-- cspell: ignore mmclassification -->
+
```
Comment (kenji-miyake) [inline perception/traffic_light_classifier/README.md:2]: https://github.com/tier4/autoware-spell-check-dict/pull/506

```suggestion
```

File: perception/traffic_light_classifier/README.md (hunk 1 above)
Comment (Shin-kyoto) [reply perception/traffic_light_classifier/README.md]: Thank you for https://github.com/tier4/autoware-spell-check-dict/commit/de0ea2b7f44992e986137ccac54eb7f428e882d2

File: perception/traffic_light_classifier/package.xml [hunk 3]
```diff
@@ -5,6 +5,7 @@
   <version>0.1.0</version>
   <description>The traffic_light_classifier package</description>
   <maintainer email="yukihiro.saito@tier4.jp">Yukihiro Saito</maintainer>
+  <maintainer email="shunsuke.miura@tier4.jp">Shunsuke Miura</maintainer>
```
Comment (kenji-miyake) [inline perception/traffic_light_classifier/package.xml:8]: @miursh Added you.

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**